mypyc/test-data/run-classes.test

```diff
@@ -1636,3 +1636,56 @@ from native import A, B, C
 a = A()
 b = B()
 c = C()
+
+[case testAttrOverrides]
+from typing import Any
+
+class Proxy:
+    def __setattr__(self, name:str, value: Any) -> None:
```

Comment:
Use an unboxed type such as `int` for `value`, so that you'll also test that argument unboxing works correctly.

In addition to `None` return type, test the `Any` return type.